config/rbac/role.yaml

```diff
@@ -5,6 +5,16 @@ metadata:
   creationTimestamp: null
```

Comment:
The changes in this file are shadowed by this: https://github.com/kairos-io/osbuilder/blob/3b33562fc324c29c16ec7b54cfd17c5c3ecafe2f/config/rbac/kustomization.yaml#L21

not sure if that affects the helm-charts too but it happens if you deploy with `kubectl apply -k config/default`

Reply:
my mistake, it's a strategic merge. The problem was different (see my other comment below).

Reply:
but it does shadow the jobs listing:

```
 User "system:serviceaccount:osartifactbuilder-operator-system:osartifactbuilder-operator-controller-manager" cannot list resource "jobs" in API group "batch" at the cluster scope
```

Reply:
I think this has been resolved with the latest changes, feel free to unresolve if not.